README.md

```diff
@@ -18,10 +20,18 @@ Prerequisites: A GitHub account (for logging into Gitpod)
 * After opening the tool via the notification, the interface loads in a few seconds.
 * Now you're ready to try the trace viewer!
     * Head to the trace viewer tab in the left side menu to get started.
-    * The tool is already loaded with example traces from a set of [Trace Visualisation Labs](https://github.com/tuxology/tracevizlab), so no need to hunt for your own.
+    * The tool is loaded with the example traces from a set of [Trace Visualisation Labs](https://github.com/tuxology/tracevizlab). To analyse your own traces, [download the application](https://github.com/theia-ide/theia-trace-extension#download-the-application) for Linux.
 
 ![gitpod-live-demo-setup](https://raw.githubusercontent.com/theia-ide/theia-trace-extension/master/doc/images/gitpod-live-demo-setup.gif)
 
+## Download the application 
+If you'd like to explore your own traces, you can **[download the application here](https://www.dorsal.polymtl.ca/files/other/electron-theia-trace-example-app-0.0.1.AppImage)!**
+
+* **Prerequisite: Java** (required since this tool reuses the [Eclipse Trace Compass server](https://download.eclipse.org/tracecompass.incubator/trace-server/rcp/?d) which runs on Java)
```

Comment:
Do we need to specify the version of Java? (We should specify if the versions people are most likely to choose/have will not work)

Reply:
Maybe this is another argument for the prerequisite eventually being Docker instead of Java? Would Docker be less picky about the version?

Reply:
@arfio ping on these questions

Reply:
Docker would be less picky. But I think for linux users, installing java is easier, given the proper command. And yes, Trace Compass requires java 11.

Reply:
Ok then could you change it to say: "Prerequisite: Java 11"?

Reply:
Ping on updating the line to:
**Prerequisite: Java 11** (required since this tool reuses the [Eclipse Trace Compass server](https://download.eclipse.org/tracecompass.incubator/trace-server/rcp/?d) which runs on Java)